MovieGallery/Pages/Search.cshtml

```diff
@@ -71,7 +73,7 @@
                     </button>
                 </h5>
             </div>
-            <div id="collapseShows" class="collapse" aria-labelledby="movies" data-parent="#accordion">
+            <div id="collapseShows" class="collapse" aria-labelledby="shows" data-parent="#accordion">
```

Comment:
In search.cshtml page on line 76 aria-label "movies" is incorrectly labelled, so to make it meaningful I labelled it correctly with "shows".
https://www.geeksforgeeks.org/best-practices-of-object-oriented-programming-oop/